src/reynir_correct/pattern.py

```diff
@@ -1440,34 +1465,35 @@ def create_patterns(cls) -> None:
                     None,
                 )
             )
-
-            # Catch "Jón leggur hann af velli."
+            # Catch "Það hefur ekki markað upphafið af því."
             p.append(
                 (
-                    "leggja",  # Trigger lemma for this pattern
-                    "VP > { VP > { VP > { 'leggja' } } PP > { P > { 'af' } NP > { 'völlur' } } }",
-                    cls.wrong_preposition_leggja_af,
+                    "upphefja",  # Trigger lemma for this pattern
```

Comment:
Again here, careful with the trigger lemma. "Upphaf" should at least be one of the trigger lemmas.

Reply:
Done!